app/views/users/profile.html.erb

```diff
@@ -0,0 +1,20 @@
+<h1 class="mb-4 text-light"> <%= @user.name %> </h1>
+
+<div class="row">
+  <div class="col-md mb-4">
+    <p class="text-light"> <%= pluralize(@user.created_events.count, 'event').upcase %> CREATED</p>
+    <div class="list-group mt-3">
+        <% @user.created_events.each do |event| %>
+          <%= link_to event.title, event_path(event), class: "list-group-item list-group-item-action" %>
+        <% end %>
```

Comment:
Let's use the @created_events variable defined at the controller instead of getting them directly from @user.